docs/source/release_procedure.org

```diff
@@ -0,0 +1,70 @@
+
+* Internal releases
+
+If a particular build does not have any requisite support from the major SDKs, and/or contains a severe dysfunction that cannot be amended in a timely fashion, the technical lead has discretion to declare the release internal during the pre-release synchronisation meeting. 
+
+* Release schedule
+
+The schedule is left up to the technical lead's discretion. However, the rule-of-thumb as to schedule is the week containing the 28-th of each month, and releasing every month.
+
+The release can be postponed if within reasonable time a highly requested feature can be implemented. If the initial estimate to the feature's completion proves insufficient, rather than extending the release further, the release is to exclude said feature. This can be referred to as a /stike one/ policy.
+
+The release can be pushed earlier, to compensate for an earlier push-back, and/or to accelerate the availability of a crucial feature within reason.
+
+Pushing the release forwards or backwards to fit around deadlines of other projects is discouraged, but is left up to the technical lead's discretion.
+
+* Development
+
+The Iroha ledger is to accumulate committed changes into the =iroha2-dev= branch (until the announcement of the deprecation of Iroha v1, and renaming of the branches to fit the new pecking order). All commits must be signed-off (for reasons of legal liability), and all merged commits must be merged with an "all green" continuous integration test report. Merging with unsatisfied requirements is left up to the technical lead's discretion, with them being responsible for any breakage that might ensue.
+
+The development is to be conducted in accordance with the [[../../CONTRIBUTING.md]] document. Employees of Soramitsu are given the option to work on a separate branch upstream in lieu of working from a personal fork (which is Hyperledger's main recommended method). The commit message format should not be overly verbose: commits that address several issues must be broken up into several (unless the opposite is more practical because of time pressure). The technical lead is given sufficient discretion to determine and change the commit messages to suit the contents of the commit, and all employees are to provide their verbal or written consent to this. In the event that the author of a commit has used an alternate email that is verifiably theirs (or is provided with a GitHub-sponsored email), the technical lead may manually override the DCO check.
+
+* Release process
+
+A week before the release window is to open, the project manager is to conduct a pre-release sync meeting. The technical lead's responsibility is to then appraise all present of the main API-breaking changes that might require attention. In preparation, the technical lead is encouraged to walk through the differential of the current =iroha2-stable= branch and =iroha2-dev= to find any changes in =schema.json= as well as other tell-tale signs of API breakage.
```

Comment:
ah, it's actually defined, but since it's internal in relation to the community, let's specify that?